index.bs

```diff
@@ -841,16 +841,35 @@ The [=effective origin=] of an {{XRSpace}} can only be observed in the coordinat
 
 <div class="algorithm" data-algorithm="populate-the-pose">
 
-To <dfn>populate the pose</dfn> of a {{XRSpace}} |space| in an  {{XRSpace}} |baseSpace| at the time represented by a {{XRFrame}} |frame| into an {{XRPose}} |pose|, the user agent MUST run the following steps:
+To <dfn>populate the pose</dfn> of an {{XRSpace}} |space| in an {{XRSpace}} |baseSpace| at the time represented by an {{XRFrame}} |frame| into an {{XRPose}} |pose|, the user agent MUST run the following steps:
 
-  1. 1. If |frame|'s [=active=] boolean is <code>false</code>, throw an {{InvalidStateError}} and abort these steps.
+  1. If |frame|'s [=active=] boolean is <code>false</code>, throw an {{InvalidStateError}} and abort these steps.
   1. Let |session| be |frame|'s {{XRFrame/session}} object.
   1. If |space|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
   1. If |baseSpace|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
-  1. If |baseSpace|'s pose cannot be determined relative to |space| at the time represented by |frame|, set |pose| to <code>null</code>.
   1. Let |transform| be |pose|'s {{XRPose/transform}}.
-  1. Set |transform|'s {{XRRigidTransform/position}} to the location of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
-  1. Set |transform|'s {{XRRigidTransform/orientation}} to the orientation of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
+  1. Query the [=/XR device=]'s tracking system for |space|'s pose relative to |baseSpace| at the time represented by |frame|, then perform the following steps:
+    <dl class="switch">
+      <dt> If the tracking system provides a [=6DoF=] pose:
+        <dd> Set |transform|'s {{XRRigidTransform/orientation}} to the orientation of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
+        <dd> Set |transform|'s {{XRRigidTransform/position}} to the position of |space|'s [=effective origin=] in |baseSpace|'s [=coordinate system=].
+        <dd> Set |pose|'s {{XRPose/emulatedPosition}} to <code>false</code>.
+
+      <dt> Else if the tracking system provides a [=3DoF=] pose or indicates that the pose position is emulated:
```

Comment:
Nit: From what I remember with the Windows MR APIs, the function call that gets you a 6DOF pose will fail if it cannot be retrieved, and a different function call can then be used to get the 3DOF pose. Would it make sense instead to say "Else if the tracking system can provide"?  Or am I overthinking this?

Reply:
This is about whether a pose is available at this very moment, not whether the device is capable of such poses.  For example, an entity that is only visually tracked may update neither its position nor its orientation when unobserved, even if "the tracking system can provide" better poses generally.

To your specific recollection here, we do make your life difficult in the Windows MR APIs by having one reference space that provides a full 6DoF pose or nothing ([`SpatialStationaryFrameOfReference`](https://docs.microsoft.com/en-us/uwp/api/Windows.Perception.Spatial.SpatialStationaryFrameOfReference)), and then a second reference space that reliably provides a 3DoF pose all the time ([`SpatialLocatorAttachedFrameOfReference`](https://docs.microsoft.com/en-us/uwp/api/Windows.Perception.Spatial.SpatialLocatorAttachedFrameOfReference)), leaving it to the app to correlate the two during tracking loss.  This was a design mistake and I'm glad both WebXR and OpenXR take care of this for the app now :smile:

I think I'd just handwave all this under "tracking system provides a 3DoF pose" - it so happens that a UA may sometimes get that provided 3DoF pose from one mechanism or another.